src/ApplicationStatus.hpp

```diff
@@ -68,6 +68,7 @@ namespace geopm
             /// @brief Set the hash of the region currently running on
             ///        a CPU.
             virtual void set_hash(int cpu_idx, uint64_t hash) = 0;
```

Comment:
Is there any reason to keep the old ```set_hash``` API?  Is it called anywhere else?

Reply:
Not really.  Only if we think there will be a case were we want set the hash but not the hint int the future.  The ApplicationStatus is intended to be an abstraction of the shared memory structure it manipulates.  I'm okay with dropping the method, it will increase our code coverage fraction, since this is no longer covered by the unit test.

Reply:
Top of the stack removes this method